### PR TITLE
fix the jcommander osgi header

### PR DESCRIPTION
**Pull request by missedone**
----
relate build failure:
[ERROR] Cannot resolve project dependencies:
[ERROR]   Software being installed: org.testng.eclipse 6.13.0.qualifier
[ERROR]   Missing requirement: org.testng 6.13.1.r201712040202 requires 'bundle com.beust.jcommander [1.7.0,2.0.0)' but it could not be found

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -90,6 +90,15 @@ platform {
         }
     }
 
+    bundle "com.beust:jcommander:1.72", {
+        bnd {
+            // override/set the symbolic name
+            symbolicName = 'com.beust.jcommander'
```
Comment (missedone) [inline build.gradle]: @cbeust , the generated osgi header of jcommander 1.72 is:
```
Manifest-Version: 1.0
Bundle-Description: A Java library to parse command line options
Bundle-License: http://www.apache.org/licenses/LICENSE-2.0
Export-Package: com.beust.jcommander.converters;version="1.72",com.beu
 st.jcommander.defaultprovider;version="1.72",com.beust.jcommander;ver
 sion="1.72",com.beust.jcommander.internal;version="1.72",com.beust.jc
 ommander.validators;version="1.72"
Build-Date: 2017-05-15
# Nick: Bundle-SymbolicName should be com.beust.jcommander
Bundle-SymbolicName: jcommander
Bundle-Version: 1.72
Bundle-Name: com.beust.jcommander
Bundle-ManifestVersion: 2
Bnd-LastModified: 1494874807778
Created-By: 1.8.0_111 (Oracle Corporation)
Tool: Bnd-2.4.0.201411031534
```

Comment (missedone) [reply build.gradle]: this is a workaround, will remove this once it's fixed in jcommander.

**Comment by missedone** [comment]
@cbeust , please apporve this PR so that I can merge, thx

**Comment by cbeust** [comment]
Sorry, I thought I already did :)

**Pull request merged**